ajax/libs/protovis/package.json

```diff
@@ -12,5 +12,18 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/mbostock/protovis"
+  },
+  "license": "BSD",
```

Comment:
You can find the license link in http://protovis.org/ is BSD-2-Clause.
cdnjs needs `license` to be SPDX identifier. So let's modify this to `BSD-2-Clause`. 
You can refer https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json or https://spdx.org/licenses/.